src/events/ReadyEvent.ts

```diff
@@ -59,9 +61,33 @@ export async function onReadyEvent(): Promise<void> {
 
             guildDoc.activeRaids.forEach(async raid => {
                 await RaidInstance.createNewLivingInstance(guildDoc, raid);
+                activeRaidIds.push(raid.raidId);
             });
         })
     ]);
 
+    LOGGER.info("Clearing unused feedback channels");
+    await Promise.all(guildDocs.map(doc => {
+        const fbChannel = Bot.BotInstance.client.channels.cache.get(doc.channels.raids.leaderFeedbackChannelId);
+        const storageChannel = Bot.BotInstance.client.channels.cache.get(doc.channels.storageChannelId);
+        if (fbChannel?.isText()) {
+            // we know it's in a guild, and we always know parent is category type
+            const parent = (fbChannel as GuildTextBasedChannel).parent as CategoryChannel;
+            parent.children.map(channel => {
+                if (!channel.isText()) return;
+                if (channel.id === fbChannel.id) return;
+
+                const topicUuid = (channel as TextChannel).topic?.split(" ")[0];
+                if (topicUuid && !activeRaidIds.includes(topicUuid)) {
+                    if (storageChannel) {
+                        RaidInstance.compileDeadFeedbackHistory(channel as TextChannel, storageChannel as TextChannel);
+                    } else {
+                        channel.delete();
```

Comment:
Use `GlobalUtilities.tryExecuteAsync()`, or something similar, to catch any potential issues. Also log any potential issues.